src/grpcbox_acceptor.erl

```diff
@@ -40,8 +42,25 @@ acceptor_terminate(Reason, _) ->
     % accept failed.
     exit(Reason).
 
-connection_allowed(_PoolName, unlimited) ->
-    true;
-connection_allowed(PoolName, MaxConns) ->
-    grpcbox_pool:connection_count(PoolName) =< MaxConns .
+connection_allowed(PeerName, PoolName, unlimited) ->
+    check_peer_allowed(PeerName, PoolName);
+connection_allowed(PeerName, PoolName, MaxConns) ->
+    grpcbox_pool:connection_count(PoolName) =< MaxConns andalso
+    check_peer_allowed(PeerName, PoolName).
+
+check_peer_allowed(PeerName, PoolName) ->
+    case throttle:check(PoolName, PeerName) of
+        {ok, _, _} ->
+            check_any_allowed(PoolName);
+        rate_not_set ->
+            check_any_allowed(PoolName);
+        {limit_exceeded, _, _} -> false
+    end.
+
+check_any_allowed(PoolName) ->
+    case throttle:check({PoolName, all}, all) of
```

Comment:
what is 'all' representing here in the scope ? on the setup you are specifying `any`

Reply:
confirmed it was a type...defaulted to using `any`